src/commands/sns_canisters_summary.rs

```diff
@@ -0,0 +1,39 @@
+use crate::lib::signing::{sign_ingress_with_request_status_query, IngressWithRequestId};
+use crate::lib::{AnyhowResult, TargetCanister};
+use crate::SnsCanisterIds;
+use candid::Encode;
+use clap::Parser;
+use ic_base_types::{CanisterId, PrincipalId};
+use std::str::FromStr;
+
+#[derive(Parser, Debug)]
+#[clap(about, version, author)]
+pub struct SnsCanistersSummaryOps {
```

Comment:
Please add a `///` style comment at the top of this method (you can copy the one you write in src/commands/mod.rs. Each command written so far has kept with this convention and I think we should continue that.